lv_conf_template.h

```diff
@@ -1131,6 +1137,19 @@
 /** LVGL Windows backend */
 #define LV_USE_WINDOWS    0
 
+/** LVGL UEFI backend */
+#define LV_USE_UEFI 0
+#if LV_USE_UEFI
+    #define LV_UEFI_HEADERS_EDK2    0   /**< Use headers from EDK2 */
+    #define LV_UEFI_HEADERS_GNU_EFI    0    /**< Use headers from gnu-efi */
+    #define LV_UEFI_HEADERS_CUSTOM  0   /**< Use custom headers */
```

Comment:
As these are mutually exclusive what about:

```suggestion
    #define LV_USE_UEFI_INCLUDE    <efi.h>
```
And create a wrapper for the `EDK2` includes.

Reply:
I replaced the defines with a single header. I wrote it that way because there are some protocol name inconsistencies between the frameworks, but they should be found by the checks in lv_uefi.h.